src/components/Sidebar/readme.md

```diff
@@ -48,3 +48,64 @@ class SimpleSidebar extends React.Component {
     </SideBarContainer>
 </div>
 ```
+
+##### Sidebar with dynamic content
+
+```js
+import React from 'react';
+import { Sidebar, SidebarItem } from 'react-rainbow-components';
+import styled from 'styled-components';
+import DynamicContent from './dynamicContent';
+import StyledDynamicContent from './styled/dynamicontent';
```

Comment:
We should not create new files for the examples, all code should be here.
We exclude examples with markdown files from the distribution files since they are not needed in order to use the library.
Note that these extra files will be distributed in the npm package and will add unnecessary overhead to the library size.